bin/common.ml

```diff
@@ -259,9 +259,12 @@ let term =
           `Error
             (true, "--dev is no longer accepted as it is now the default.")
       and profile =
+        let doc =
+          "Build profile. dev if unspecified or release if -p is set." in
```

Comment:
This isn't quite clear as to precedence. What happens if `DUNE_PROFILE=dev` and `dune build -p foo` is called?

Reply:
It should be an error. Just like passing `--profile` and `-p` simultaneously.